### DISCUSSION TITLE
Default sdk region is not working when using gov cloud account

### DISCUSSION THREAD

**Discussion by adelcheva** [aws/aws-sdk-java#2762] [General]
Currently when you try to make an SDK call, you need to have AwsAsyncClientBuilder, which needs to have the region passed, but when you select the marked in the SDK as DEFAULT_REGION this doesn't work for the government cloud account credentials. Do you have a way to go around this, or we need to rework that in our usages somehow? The hard part is that we don't know the exact region in every place.

**Comment by debora-ito** [comment]
For Gov Cloud accounts, you'd need to set the client region to the gov cloud regions `Regions.GovCloud` or `Regions.US_GOV_EAST_1`. I don't think DEFAULT_REGION will work in this case.
